core/src/client/rpc.rs

```diff
@@ -74,6 +74,8 @@ trait EthRpc<TX: TransactionResponse + RpcObject, TXR: RpcObject, R: ReceiptResp
     ) -> Result<U64, ErrorObjectOwned>;
     #[method(name = "getCode")]
     async fn get_code(&self, address: Address, block: BlockTag) -> Result<Bytes, ErrorObjectOwned>;
+    #[method(name = "getClientVersion")]
```

Comment:
Oh sorry one last thing. This should be `web3_clientVersion`, not `eth_getClientVersion`. You can see how we do that with the net vs eth namespaces we already support.

Reply:
Pushed these changes in latest commit